bouncer/views.py

```diff
@@ -66,11 +75,12 @@ def index(request):
         location=request.registry.settings["hypothesis_url"])
 
 
-@view.notfound_view_config(
-    renderer='bouncer:templates/notfound.html.jinja2')
-def notfound(request):
-    request.response.status_int = 404
-    return {}
+@view.view_config(
+    context=httpexceptions.HTTPException,
+    renderer='bouncer:templates/error.html.jinja2')
+def httpexception(exc, request):
+    request.response.status_int = exc.status_int
+    return {"message": exc.message}
```

Comment:
Does this not raise a deprecation warning? `BaseException.message` has been deprecated for a long time. Use `str(exc)` instead.